src/PostgREST/Config.hs

```diff
@@ -43,17 +44,19 @@ import           Protolude hiding            (intercalate
 
 -- | Config file settings for the server
 data AppConfig = AppConfig {
-    configDatabase  :: Text
-  , configAnonRole  :: Text
-  , configProxyUri  :: Maybe Text
-  , configSchema    :: Text
-  , configHost      :: Text
-  , configPort      :: Int
-  , configJwtSecret :: Maybe Text
-  , configPool      :: Int
-  , configMaxRows   :: Maybe Integer
-  , configReqCheck  :: Maybe Text
-  , configQuiet     :: Bool
+    configDatabase          :: Text
+  , configAnonRole          :: Text
+  , configProxyUri          :: Maybe Text
+  , configSchema            :: Text
+  , configHost              :: Text
+  , configPort              :: Int
+  , configJwtSecretOrFile   :: Maybe Text
```

Comment:
Is it possible to do everything with just `configJwtSecret`? After processing, `configJwtSecretOrFile` gets turned into Nothing. Wondering if the secret or file could be stored in `configJwtSecret` and then this data field be overwritten by the properly encoded value?